libs/common/include/helpers/mathFuncs.h

```diff
@@ -79,4 +81,19 @@ constexpr T inverseLerp(const T startVal, const T endVal, const T value) noexcep
 {
     return (value - startVal) / (endVal - startVal);
 }
+
+// TODO can be constexpr in C++20
+/// Arithmetically round floating point values to integers
+template<typename IntType, typename FloatType,
+         std::enable_if_t<std::is_integral<IntType>::value && std::is_floating_point<FloatType>::value, int> = 0>
+inline IntType iround(const FloatType val) noexcept
+{
+    RTTR_Assert(!std::isnan(val));
+    RTTR_Assert(!std::isinf(val));
```

Comment:
Good idea but I guess a combined check is enough:
```suggestion
    RTTR_Assert(!std::isfinite(val));
```